### PR TITLE
Run JuliaFormatter.format()

### PR DESCRIPTION
**Pull request by LilithHafner**
Using JuliaFormatter v1.0.49

### CODE REVIEW

File: src/callbacks.jl [hunk 1]
```diff
@@ -612,7 +612,7 @@ end
         integrator.u_modified = true
         callback.affect!(integrator)
         if integrator.u_modified
-            reeval_internals_due_to_modification!(integrator, #=continuous_modification=# false)
+            reeval_internals_due_to_modification!(integrator, false) #=continuous_modification=#
```
Comment (oscardssmith) [inline src/callbacks.jl:615]: this is incorrect. This comment was specifically to refer to the `false`

Comment (LilithHafner) [reply src/callbacks.jl:615]: I'll report this upstream. Until then, perhaps we should simply close this PR and leave format-check broken.

Comment (ChrisRackauckas) [reply src/callbacks.jl:615]: ehh we can just change that comment, we can report upstream but let's just get the green check mark done.

Comment (ChrisRackauckas) [inline src/callbacks.jl:615]:
```suggestion
            reeval_internals_due_to_modification!(integrator, false) # continuous_modification=false
```

**Pull request merged**